.ruby-version

```diff
@@ -1 +1 @@
-2.7.2
+3.0.6
```

Comment:
I tried to go up to `Ruby 3.1.4` but I got some errors with the new `OpenSSL v3` to install Rubies but we should be good for now I think.

<img width="1070" alt="Screenshot 2023-09-27 at 23 04 51" src="https://github.com/challengepost/profiles-pairing-app/assets/13169164/f0b7e1e1-65c5-4013-aac4-ecddef675698">